cypress/e2e/auth.cy.ts

```diff
@@ -1,11 +1,35 @@
 describe("Authentication tests", () => {
+    const email: string = Cypress.env("TEST_USER");
+    const password: string = Cypress.env("TEST_PASS");
+
+    const extendedTimeout = { timeout: 10000 };
+
+    const buttonReachablePaths = ["/", "/clients", "/lists", "/calendar"];
+    const allPaths = [...buttonReachablePaths, "/clients/add"];
+
     it("Redirected to login page", () => {
-        for (const url of ["/", "/clients", "/lists", "/calendar", "/login"]) {
+        for (const url of allPaths) {
             cy.visit(url);
             cy.url().should("include", "/login");
         }
     });
 
+    it("Redirected to clients after login", () => {
+        cy.visit("/login");
+
+        cy.get("#login-panel[data-loaded='true']", extendedTimeout).should("exist");
+
+        cy.get("input#email").type(email);
+        cy.get("input#password").type(password);
+
+        cy.get("#email").should("have.value", email);
```

Comment:
why sometimes `#email` and sometimes `input#email`?

Reply:
can this be 'better' semantically? e.g. testing for the thing that has 'email' in the label - idk what you can access